frontend/src/app/About/Contributors.tsx

```diff
@@ -0,0 +1,96 @@
+import { Globe } from 'iconoir-react';
+import { contributorStructure } from './index';
+import styles from "./Contributors.module.scss"
+
+interface ContributorsProps {
+	currContributors: contributorStructure[];
+	alumniContributors: contributorStructure[];
+}
+
+
+const Contributors: React.FC<ContributorsProps> = ({ currContributors, alumniContributors }) => {
```

Comment:
When you are writing component, I would always use syntax like:
```ts
export default function Contributors({ currContributors, ... }: ContributorsProps) {
   ...
}
```
When you export anonymous functions as components, they aren't properly handled by React developer tools and you need to define `Contributors.displayName = "Contributors"` before you export, but I find it much easier to export as I showed above.